ci-operator/jobs/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master-presubmits.yaml

```diff
@@ -1338,21 +1338,21 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-ipi-ovn-ipv6,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
Was it intended to run virtualmedia on all etcd master PR's?

Reply:
I'm not sure why this happened, I changed the job being used from "e2e-metal-ipi-serial-ipv4" to "e2e-metal-ipi-serial-virtualmedia" and ran "make jobs", I wouldn't have expected this change, looking into it